actions/destinations/search.go

```diff
@@ -32,12 +33,21 @@ func (a *Action) search(c *gin.Context) {
 		return
 	}
 
+	var destFilters models.DestinationFilters
```

Comment:
This model should be extracted from request body instead of conditions map.

Also it should be included in Swagger